ST_structs.h

```diff
@@ -1,333 +1,721 @@
+      /** \brief millimeters of extra resources. */
   IntUS mm_extra_res,
+      /** \brief Species[indv->myspecies] is the species this individual belongs to.
+       * \sa Species  */
       myspecies;
-  RealF relsize,      /* relative to full-sized individual -- 0-1.0) */
-       grp_res_prop,  /* prop'l contribution this indiv makes to group relsize */
-       res_required, /* resources required, biomass of the individual */
-       res_avail,    /* resource available */
-       res_extra,    /* resource applied to superficial growth */
-       pr,           /* ratio of resources required to amt available */
-       growthrate;   /* actual growth rate*/
-  struct indiv_ann_st *Next, *Prev;  /* facility for linked list 8/3/01 */
+      /** \brief Size relative to a full-sized individual. */
+  RealF relsize,
+       /** \brief proportion of total group relsize belonging to this individual. */
+       grp_res_prop,
+       /** \brief resources required, biomass of the individual. 
+        * \sa rgroup_ResPartIndiv */
+       res_required,
+       /** \brief Resources available. */
+       res_avail,
+       /** \brief Resources applied to superficial growth. */
+       res_extra,
+       /** \brief Ratio of resources required to amt available. */
+       pr,
+       /** \brief Actual growth rate .
+        * \sa rgroup_Grow() */
+       growthrate;
+        /** \brief Allows for a doubly-linked list of individuals. Implemented in Species.
+         * \sa Species */
+  struct indiv_ann_st *Next, 
+        /** \brief Allows for a doubly-linked list of individuals. Implemented in Species.
+         * \sa Species */
+                  *Prev;
 };
 
-/* structure for species with some annuals-only variables thrown in */
+/** 
+ * \brief Holds all species-specific information.
+ * 
+ * This struct contains constants and variables that differ between
+ * species. \ref Species is a global array of these structs.
+ * 
+ * \sa Species
+ * 
+ * \ingroup SPECIES
+ */
 struct species_st {
 
   /**** Quantities that can change during model runs *****/
 
-  SppIndex est_count;  /* number of individuals established (growing)*/
-  IntUS *kills,        /* ptr to array of # indivs killed by age. index=age. */
-        estabs,        /* number of individuals established in iter */
-        *seedprod,    /* annuals: array of previous years' seed production (size = viable_yrs)*/
+      /** \brief Number of individuals established (growing) */
+  SppIndex est_count;
+      /** \brief Pointer to array of individuals killed, indexed by age at death. */
+  IntUS *kills,
+      /** \brief Number of individuals established in current iteration. */
+        estabs,
+      /** \brief Specific to annuals. Array of seeds produced indexed by year. 
+       * \sa ST_seedDispersal.c */
+        *seedprod,
+      /** \brief not sure what this does yet.
+       * \sa ST_seedDispersal.c */
          seedbank,
```

Comment:
as far as I can tell it is not used

Reply:
@chaukap should we open an issue for this as well to investigate what the seedbank does if anything?